src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig

```diff
@@ -81,6 +81,9 @@
         {{ form_widget(form) }}
         {{ form_errors(form, {'attr': {'fieldError': true}}) }}
       </div>
+      {% if attribute(form.parent, form.vars.name) is defined and attribute(form.parent, form.vars.name).vars.multistore_dropdown != false %}
+        {{ attribute(form.parent, form.vars.name).vars.multistore_dropdown | raw }}
```

Comment:
⚠️  a `| raw` in Twig is a sensible loc 👀

Reply:
Could we replace it with `| escape(strategy)` and choose the right `strategy` https://twig.symfony.com/doc/2.x/filters/escape.html

Reply:
I'm not sure there is a corresponding strategy... the idea is to keep twig from escaping HTML 🤔 (which he does by default, `| escape('html')` .